addons/prometheus-server/resources/thanos-query.cue

```diff
@@ -3,7 +3,6 @@ package main
 thanosQuery: {
 	name: "thanos-query"
 	type: "webservice"
-	dependsOn: ["prometheus-service"]
```

Comment:
Why remove the dependsOn?

Reply:
Because there is no component named "prometheus-service". It is a legacy component name that is not in use and should be deleted.